### DISCUSSION TITLE
Kyty using Command Line?

### DISCUSSION THREAD

**Discussion by GeorgeSpiggott** [InoriRus/Kyty#64] [Q&A] [answered]
Is it possible to boot directly into games using the command line or is is necessary to use the launcher app?

**Comment by InoriRus** [comment, marked as answer]
Yes. It is possible. The emulator itself is just a Lua interpreter. You can run fc_script.exe by passing correct lua script as argument.
Launcher is just a tool to help you generate the lua script.

**Comment by GeorgeSpiggott** [comment]
Thanks, it took me a while to figure out that fc_script requires the whole path of the Lua. The following works fine as a .bat file.

<location of>\Kyty-v0.2.0\fc_script.exe <location of>\Kyty-v0.2.0\kyty_run.lua   

kyty_run.lua will allow you to play the last game that was run via Launcher. I'll look into something more game specific and cleaner for EmulationStation when I can get something to run.
